addons/binding/org.openhab.binding.smartmeter/src/test/java/org/openhab/binding/meterreader/TestMeterReading.java

```diff
@@ -0,0 +1,77 @@
+package org.openhab.binding.smartmeter;
```

Comment:
Normally, tests should be put into a dedicated test fragment, not in the main bundle.
But it seems that those classes are not meant to be (automatically executed) unit tests, but rather helper classes for performing some manual testing, is this correct? If this is the case, I'm ok to leave them here. (you could add the license header to them as well, though).

Reply:
I've put them now in a dedicated test fragment and made them automatically executed unit tests.